packages/@o3r/rules-engine/package.json

```diff
@@ -122,6 +122,7 @@
     "@o3r/testing": "workspace:^",
     "@schematics/angular": "~18.2.0",
     "@stylistic/eslint-plugin": "~2.7.0",
+    "@swc/helpers": "~0.5.0",
```

Comment:
Why it is now needed in this package?
(same question for `@o3r/schematics`)

Reply:
Nx uses swc to compile the executor inside the monorepo, I think

Reply:
Today the executor are compiled with `tsc`.
Does it mean that the node-ts based solution to executor the executor in TS is not needed any more?

If so we may need to remove the `index.js`.
Otherwise I don't see the reason for adding this dep :S

Reply:
Actually, I've already removed all the `index.js` or did I miss some?